### PR TITLE
fix(plugin): fix options assignment

### PR DESCRIPTION
**Pull request by ThornWalli**
Hello @peterramsing,

still have a problem applying the user setting via an options object.

Here is a `Bitwise OR` used 😉 (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_OR)
```
 let runSettings = assign({}, defaultSettings, settings | {});
```

But we need a `Logical OR` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR)

```
 let runSettings = assign({}, defaultSettings, settings || {});
```

Thanks!

### CODE REVIEW

**Comment by peterramsing** [comment]
Well shucks. I'm glad I got pulled away and forgot to release that night. Would be worth writing some tests for this.

**Comment by peterramsing** [comment]
PS: Thank you for the help!

**Comment by peterramsing** [comment]
Current tests are running - then either you're welcome to get a test that fails without this code so we can make sure we don't regress *or* I'll snag some time to do it. I've added a flag to this on my OmniFocus project so it's surfaced better for me.

**Comment by ThornWalli** [comment]
Created a test :)

File: test/lost-plugin-options.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    `;
+    const result = await postcss([
+      lost({
+        gutter: '15px',
+        flexbox: 'flex',
+        cycle: 'auto'
+      })
+    ])
+      .process(input);
+      expect(output).to.equal(result.css);
```
Comment (peterramsing) [inline test/lost-plugin-options.js]:
```suggestion
      expect(result.css).to.equal(output);
```
It's minor, but this reads better when reading errors.

What it should read like when this test fails:
Screen Shot 2022-06-29 at 21 56 23

**Review by peterramsing — CHANGES_REQUESTED** [review]
One tiny nit to make the tests more readable if they fail.

Thanks for doing this!

**Comment by peterramsing** [comment]
doh - linting rules are failing the CI. 👀

**Comment by peterramsing** [comment]
...I want to look at this further but I have to get some sleep. I have some time tomorrow slated to look at this. Not sure if I want to change the eslint or adapt the test. Also, it looks like the coverage drops on `lost.js` on `ln50` so that needs a quick glance. Not sure how that happened. 🤔

**Comment by peterramsing** [comment]
@ThornWalli, if you pull the latest from `master` the tests should pass.

---

I'm not sure why the coverage fell a bit. Gonna poke on that a bit more.

**Comment by peterramsing** [comment]
Weird - coverage didn't change, now. <shakes fist at computer>
--
Looks good to me! Thanks!

**Pull request merged**